sdk/communication/azure-communication-jobrouter/pom.xml

```diff
@@ -94,6 +94,12 @@
       <version>5.9.3</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-params;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.14.2</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <scope>test</scope>
```

Comment:
This should not be test scope. This should be compile scope dependency. We don't want to test with one version of netty and release with a different version (through azure-communication-common)

Reply:
Actually, the right fix is to update the tag of azure-communication-common to use `current` as they are in the same service group - `<!-- {x-version-update;com.azure:azure-communication-common;current} -->`

Reply:
> Actually, the right fix is to update the tag of azure-communication-common to use `current` as they are in the same service group - `<!-- {x-version-update;com.azure:azure-communication-common;current} -->`

@srnagar This would be correct if all of the azure-communication libraries were part of a single pipeline like sdk/core but they're not. The azure-communication libraries are all owned by separate teams and each library has its own individual pipeline. Things only use current if they're building and releasing as part of the same pipeline, in this case azure-communication-common's version being dependency, since it's released separately, is correct.

Reply:
Thanks @JimSuplizio  for the clarification. We should then add `azure-core-netty` as a compile time dependency to not have different versions of core and netty that aren't compatible.

Reply:
Changed to `compile`.

I assume the implicit rule for communication would be that the common lib must always be released first, before any (at least for GA) lib release new version?